tests/test_models.py

```diff
@@ -717,9 +717,22 @@ def test_validate_audit_action_audit(self):
     def test_validate_audit_action_ignore(self):
         self._func(audit_action=AuditAction.IGNORE)  # does not raise
 
-    def test_validate_audit_action_raises_unsetauditactionerror(self):
-        with self.assertRaises(UnsetAuditActionError):
+    def test_validate_audit_action_raise(self):
+        with self.assertRaises(UnsetAuditActionError) as test:
+            self._func(audit_action=AuditAction.RAISE)
+        self.assertEqual(
+            str(test.exception),
+            "TestValidateAuditAction._func() requires an audit action",
+        ),
+
+    def test_validate_audit_action_requires_keyword_argument(self):
+        with self.assertRaises(UnsetAuditActionError) as test:
             self._func()
```

Comment:
Wouldn't you want the test to attempt to pass an audit_action as a positional arg here?

Reply:
Offlined with Joel and realized that this is testing the validator, and behavior of passing nothing vs any positional arg will be the same so nevermind!